### ISSUE TITLE
Build & Run gives "-1" Warning on new install, new project.

### ISSUE THREAD

**Issue by NatePlays95** [chrismaltby/gb-studio#1088]
**Describe the bug**
Seems any project I try to build and run under my installation of GBStudio 3 will give me this very vague "-1" warning, and nothing happens. It appears when I open a previous project, when I try on any of the two sample projects, and on the blank project as well.
This is my first time using GBStudio ever, so the only thing I even got to do with it was changing the player sprite (which shouldn't matter) before I got stuck with these errors.

**To Reproduce**
1. Open Software
2. Create a new project (sample or blank)
3. Click the Run button

**Expected behavior**
I expect the game to run on a small separate window after a few moments of clicking the Run button.

**Screenshots**
![image](https://user-images.githubusercontent.com/74799857/159806080-e3a5a954-0c91-41bb-ad71-2ae6c7294c2f.png)
![image](https://user-images.githubusercontent.com/74799857/159806966-e3314e87-955b-4510-9746-fe5694f10b56.png)
I get these two versions at random, one with and one without including camera.c

**Platform (please complete the following information):**
 - OS: Windows 10, 64bit
 - App version 3.0.3, 64bit

**Additional context**
As of now I've tried:
- Cleaning build cache
- Restarting my laptop
- Testing on other projects, as read before
- Reinstalling via the x64 installer
- Reinstalling via the standalone x64 download (on a separate location)

I'm afraid this bug is not easily reproduceable? If it was, any normal user would notice it immediately.

**Comment by NatePlays95** [comment]
Hope this is not too aggressive on here, but I also tested Installer 3.02 and Standalone 3.01 from this repo.
This time it actually gives me message errors.
<details>
  <summary>3.02</summary>

![image](https://user-images.githubusercontent.com/74799857/159810279-bdc0e2e2-8d03-4656-b82b-a62f42be46cb.png)

</details>
<details>
  <summary>3.01</summary>

![image](https://user-images.githubusercontent.com/74799857/159810663-c5485443-20b0-441b-b30d-e0c268303189.png)

![image](https://user-images.githubusercontent.com/74799857/159810597-4972a7ed-2ead-4f9c-9450-4ea062058bf3.png)

![image](https://user-images.githubusercontent.com/74799857/159810773-08e6afff-509b-428e-8955-f35b219341c0.png)

</details>

Both of these were from creating a brand new Sample 1.0 project and directly running it.

**Comment by maxoakland** [comment]
It says there are empty source files

**Comment by NatePlays95** [comment]
Correct, but I can read that myself. From testing the 2.0 beta I could get a clue of what's happening:
![image](https://user-images.githubusercontent.com/74799857/159821928-449fd548-c356-44d6-b9dd-a02e506d68cc.png)
For some reason my Windows is running GBStudio's C compiler in ISO mode (the more strict one), and thus any useless files are being deleted. The engine then request those files again and throws errors. I'm not aware of how to change that.

My friend told me he had a similar problem and dropped the engine as well. I'm playing around with 1.0 a bit but seeing the state of bug issues here support is scarce. ~~There shouldn't even be a win32 ver on itch.io~~
Hope I'll have some motivation to keep using this if someone finds a fix, the app itself is very lovely.
